motion/los_guidance/scripts/los_guidance_euler.py

```diff
@@ -48,18 +64,49 @@ def __init__(self):
 
 
 	def updateState(self, x, y, z, u, v, w, psi, r, time):
+		"""
+		Update all state values contained in the LOS class.
+
+		Args:
+			x	  Surge; position in the direction of the x-axis.
+			y	  Sway;  position in the direction of the y-axis.
+			z	  Heave; position in the direction of the z-axis.
+
+			u	  Body fixed velocity in the x-direction.
+			v	  Body fixed velocity in the y-direction.
+			w	  Body fixed velocity in the z-direction.
+
+  			psi	  Heading angle required to reach the LOS intersection
+			  	  point.
+			r
```

Comment:
It is clear what r is - given that people have read the documentation. However, we live in a world where Dragvoll will move to Gløs, and therefore we cannot assume that the person reading this in the future will have more than one braincell ;)

Reply:
I initially thought r was the radius of acceptance, but that is given by capital R. Is this the radius from the current body position to the straight line path? Tagging @jae-hyeong-hwang  and @michoy too, in case they know :smile:

Reply:
Will approve the merge when "r" is defined. Could easily be a misspelling, as @param R is only mentioned at brief (as far as I am aware of)

Reply:
R is hardcoded as `self.R = 0.5` in the __init__ for the LOS class, so I think they are two different parameters, and I don't know if R is something that needs to be updated dynamically. However, self.r isn't used anywhere else, so I can't say for sure.

Reply:
It is possible to accept the PR right now, but make an issue about "r" that could be solved later. I think it is better that 99 % of the code is documented, compared to not allowing the merge and leaving the code undocumented

Reply:
That is fair. r, r_d and r_d_dot also appear in the backstepping controller, so I will include that too 👍

Reply:
Issue #95 now mentions this problem ^

Reply:
Sweet! Nice work